samples/ballerina-by-example/examples/passthrough/passthrough.bal

```diff
@@ -2,26 +2,26 @@ import ballerina.net.http;
 
 service<http> passthrough {
 
-    @Description {value:"Requests which contain any HTTP method will be directed to passthrough resource."}
+    endpoint<http:HttpClient> endPoint {
+        create http:HttpClient("http://localhost:9090/echo", {});
+    }
+
+    @Description {value:"Any type of HTTP requests will be dispatchd to the passthrough resource."}
```

Comment:
dispatched*
"Any type of HTTP requests"  doesn't sound meaningful to me as we need to emphasize on HTTP method specifically.